src/main/resources/db/migration/V1_11__REC_FLOW_EVENT_TABLE.sql

```diff
@@ -0,0 +1,7 @@
+drop table if exists recflowevents;
+
+CREATE TABLE recflowevents
+(
+    id serial primary key,
+    data jsonb
```

Comment:
Is it faster to query and join if there are separate columns for each data point?